### PR TITLE
Fixed Error import from cassandra

### PR DESCRIPTION
**Pull request by yershalom**
Hi,

Added import Error by mistake.
Replaced it by DriverException

### CODE REVIEW

File: redash/query_runner/cass.py [hunk 1]
```diff
@@ -7,8 +7,9 @@
 logger = logging.getLogger(__name__)
 
 try:
-    from cassandra.cluster import Cluster, Error
+    from cassandra.cluster import Cluster
     from cassandra.auth import PlainTextAuthProvider
+    from cassandra import DriverException
```
Comment (arikfr) [inline redash/query_runner/cass.py]: I think that the Cassandra driver version we have in the requirements file doesn't have this :-( Can you confirm?

Comment (arikfr) [inline redash/query_runner/cass.py]: ?

Comment (yershalom) [inline redash/query_runner/cass.py]: Hi @arikfr 
I got confused.
I didn't notice that I worked with cassandra-driver==3.4.0 on my test server.

**Comment by yershalom** [comment]
@arikfr Made a new one

**Pull request closed without merging**